packages/gen-ai/package.json

```diff
@@ -1,17 +1,20 @@
 {
   "private": true,
-  "name": "@odh-dashboard/llama-stack-modular-ui",
-  "description": "LLama Stack Modular UI plugin.",
+  "name": "@odh-dashboard/gen-ai",
+  "description": "Gen AI plugin.",
   "version": "0.0.0",
+  "exports": {
+    "./extensions": "./frontend/src/odh/extensions.ts"
+  },
   "module-federation": {
-    "name": "llamaStackModularUi",
+    "name": "genAi",
     "remoteEntry": "/remoteEntry.js",
     "authorize": true,
     "tls": false,
     "proxy": [
       {
-        "path": "/llama-stack/genai",
-        "pathRewrite": "/genai"
+        "path": "/llama-stack/gen-ai",
```

Comment:
I think this path is still correct `"path": "/llama-stack/gen-ai",` but please confirm.